types/node/index.d.ts

```diff
@@ -1782,6 +1782,9 @@ declare module "child_process" {
         stdio: [stream.Writable, stream.Readable, stream.Readable];
         pid: number;
         kill(signal?: string): void;
+        send(message: any): boolean;
+        send(message: any, callback?: (error: Error) => void): boolean;
+        send(message: any, sendHandle?: net.Socket | net.Server, options?: MessageOptions): boolean;
```

Comment:
I think this one is not needed as the next one with optional callback after options covers the `send(message, sendHandle, options)` and `send(message, sendHandle, options, cb)` use cases.
But I think the use case `send(message, sendHandle, cb)` is not yet covered.